docker/generic/start_proxy.py

```diff
@@ -869,6 +869,15 @@ def make_argparser():
         in grpc-json transcoding. This is to support HTML 2.0<https://tools.ietf.org/html/rfc1866#section-8.2.1>.
         Set this flag to true to disable this feature.
         ''')
+    parser.add_argument(
+        '--allow_colon_in_wildcard_path_segment', action='store_true',
```

Comment:
What is the default value?  If false, you are changing the default behaviors.  It may break some users for ESPv2.

Reply:
@nareddyt and I both agree it is better to enable this change by default, as:
- we should follow Google spec and the misroute is a bug we wanna fix for all users
- it also aligned with our transcoder behavior

Reply:
For  a /foo/{x=*},    /foo/bar1:bar2 used to work with x=bar1:bar2  after the change, it will be rejected.   This is a breaking change.  Prefer not to have  breaking change.